labgrid/resource/udev.py

```diff
@@ -703,3 +703,19 @@ def filter_match(self, device):
             return False
 
         return super().filter_match(device)
+
+@target_factory.reg_resource
+@attr.s(eq=False)
+class JLinkDevice(USBResource):
+    """The JLinkDevice describes an attached Segger J-Link device,
+    it is identified via USB using udev
+    """
+
+    def __attrs_post_init__(self):
+        self.match["ID_VENDOR_ID"] = "1366"
```

Comment:
Since USBDebugger now covers the same vendor & model ID as JLinkDevice, do I need to refactor to use USBDebugger instead?